src/main/scala/cromwell/binding/AstTools.scala

```diff
@@ -76,7 +76,7 @@ object AstTools {
 
       astNode match {
         case t: Terminal if t.getTerminalStr == "string" && wdlType == WdlStringType => WdlString(t.getSourceString)
-        case t: Terminal if t.getTerminalStr == "string" && wdlType == WdlFileType => WdlFile(t.getSourceString)
+        case t: Terminal if t.getTerminalStr == "string" && wdlType == WdlFileType => WdlFile(t.getSourceString, false)
```

Comment:
I think I like more the subtype route, but then it gets more complicated to test if a WdlFile is actually a glob (need isInstanceOf or collect / case ...)
Although conceptually WdlGlobFile should even be a subtype of WdlArray[WdlFile] IMO.